src/videostreamidset/DefaultVideoStreamIdSet.ts

```diff
@@ -66,4 +66,9 @@ export default class DefaultVideoStreamIdSet implements VideoStreamIdSet {
   toJSON(): number[] {
     return this.array();
   }
+
+  truncate(length: number): DefaultVideoStreamIdSet {
```

Comment:
Should the return type be VideoStreamIdSet?

Reply:
We are returning the `DefaultVideoStreamIdSet` in the clone function. so to maintain conformity, I am using the same signature here.